test/rules/assertions/validTypes.js

```diff
@@ -161,14 +161,82 @@ export default {
           }
       `,
       errors: [{
-        line: 3,
-        message: 'Syntax error in type: '
+        line: 2,
```

Comment:
This line number behavior is different than how others have been calculated. For consistency, it should be 3 here.

Reply:
Fixed this